modules/quickstart-public-cloud/pages/installation.adoc

```diff
@@ -35,7 +37,7 @@ When you have logged in to the administrator account for the first time, change
 Add the required software products to your client, and schedule a repository synchronization.
 
 
-
+ifeval::[{uyuni-content} == true]
```

Comment:
This isn't required, the endif needs to be deleted instead. It's just copypasta. I thought I took it out already, sorry.